Lib/annotationlib.py

```diff
@@ -0,0 +1,635 @@
+"""Helpers for introspecting and wrapping annotations."""
+
+import ast
+import enum
+import functools
+import sys
+import types
+
+
+class Format(enum.IntEnum):
```

Comment:
So you eventually decided not to have a global enumeration?

Reply:
It's in PEP 749 this way.